Sources/Build/LLBuildManifestBuilder.swift

```diff
@@ -506,11 +530,27 @@ extension LLBuildManifestBuilder {
     private func addCmdWithBuiltinSwiftTool(
         _ target: SwiftTargetBuildDescription,
         inputs: [Node],
-        cmdOutputs: [Node]
+        cmdOutputs: [Node],
+        mixedTarget: Bool = false
     ) throws {
         let isLibrary = target.target.type == .library || target.target.type == .test
         let cmdName = target.target.getCommandName(config: buildConfig)
 
+        var otherArguments = try target.compileArguments()
+        if mixedTarget {
```

Comment:
Should this logic be in `compileArguments`?

Reply:
Moved in f13865a. I tried to move build logic out of this file and into each `(Clang|Swift|Mixed)TargetBuildDescription` type.